mockgen/tests/plain_func/input.go

```diff
@@ -0,0 +1,5 @@
+//go:generate mockgen -destination reflect_output/mock.go github.com/golang/mock/mockgen/tests/plain_func Func
+//go:generate mockgen -source input.go -package plain_func -destination source_output/mock.go
+package plain_func
+
+type Func func(int) int
```

Comment:
Can we add some code that uses the generated output? This should help to ensure we don't break this down the road?

Reply:
I just pushed a second commit to add such a test. While there, I also moved the test to the right location. It looks like the tests were moved around at some point.